packages/backend-core/src/security/auth.ts

```diff
@@ -0,0 +1,29 @@
+import { env } from ".."
+
+export const PASSWORD_MIN_LENGTH = 8
+export const PASSWORD_MAX_LENGTH = 512
+
+export function validatePassword(
+  password: string
+): { valid: true } | { valid: false; error: string } {
+  if (env.isDev() && !env.isTest() && password) {
+    // We accept any password while on development
+    return { valid: true }
+  }
+
+  if (!password || password.length < PASSWORD_MIN_LENGTH) {
+    return {
+      valid: false,
+      error: "Password invalid. Minimum eight characters.",
```

Comment:
Can we use `PASSWORD_MIN_LENGTH` here, similar to how we use `PASSWORD_MAX_LENGTH` below?